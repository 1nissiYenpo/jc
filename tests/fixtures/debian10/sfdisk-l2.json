[{"disk":"/dev/sda","disk_size":"32 GiB","bytes":34359738368,"sectors":67108864,"disk_model":"QEMU HARDDISK","units":"sectors of 1 * 512 = 512 bytes","logical_sector_size":512,"physical_sector_size":512,"min_io_size":512,"optimal_io_size":512,"disk_label_type":"dos","disk_identifier":"0x4f1ce2bf","partitions":[{"device":"/dev/sda1","boot":true,"start":2048,"end":13260799,"sectors":13258752,"size":"6.3G","id":"83","type":"Linux"},{"device":"/dev/sda2","boot":false,"start":13262846,"end":67106815,"sectors":53843970,"size":"25.7G","id":"5","type":"Extended"},{"device":"/dev/sda5","boot":false,"start":13262848,"end":18331647,"sectors":5068800,"size":"2.4G","id":"83","type":"Linux"},{"device":"/dev/sda6","boot":false,"start":18333696,"end":20332543,"sectors":1998848,"size":"976M","id":"82","type":"Linux swap / Solaris"},{"device":"/dev/sda7","boot":false,"start":20334592,"end":21352447,"sectors":1017856,"size":"497M","id":"83","type":"Linux"},{"device":"/dev/sda8","boot":false,"start":21354496,"end":67106815,"sectors":45752320,"size":"21.8G","id":"83","type":"Linux"}]}]
