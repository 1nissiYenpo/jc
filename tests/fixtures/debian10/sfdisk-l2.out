Disk /dev/sda: 32 GiB, 34359738368 bytes, 67108864 sectors
Disk model: QEMU HARDDISK   
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x4f1ce2bf

Device     Boot    Start      End  Sectors  Size Id Type
/dev/sda1  *        2048 13260799 13258752  6.3G 83 Linux
/dev/sda2       13262846 67106815 53843970 25.7G  5 Extended
/dev/sda5       13262848 18331647  5068800  2.4G 83 Linux
/dev/sda6       18333696 20332543  1998848  976M 82 Linux swap / Solaris
/dev/sda7       20334592 21352447  1017856  497M 83 Linux
/dev/sda8       21354496 67106815 45752320 21.8G 83 Linux
